.github/workflows/pr-checker.yml

```diff
@@ -0,0 +1,97 @@
+name: PR-checker
+on: push
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: izuma-system-tests-'${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  static-checks:
+    runs-on: ubuntu-latest
+    env:
+      SUMMARY_FILE: summary.log
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: Set GitHub access token via git config
+        run: | 
+          git config --global url."https://${{ secrets.ACCESS_TOKEN }}@github.com/".insteadOf "git@github.com:"
```

Comment:
Is this needed only for the git clone?
If yes, you can use this
```
        uses: actions/checkout@v3
        with:
          repository: 
          token: ${{ secrets.ACCESS_TOKEN }}
```

Reply:
Done.